third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -16,6 +16,13 @@ import (
 	"google.golang.org/api/compute/v1"
 )
 
+const (
+	// Some fields are only supported by region IGM - these flags are used to
```

Comment:
I'm a little 👎 on using the same method as the resources diverge. While it's fine now while there's one field that's different, I'm afraid of carrying the pattern forward if more fields like this are added.

Reply:
yeah, i could go either way on splitting right now honestly